go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
 	google.golang.org/appengine v1.6.5 // indirect
 	google.golang.org/genproto v0.0.0-20200413115906-b5235f65be36 // indirect
-	google.golang.org/grpc v1.26.0
+	google.golang.org/grpc v1.27.0
```

Comment:
It seems grpc is expected to be in version 1.26. I am working in #961 to get that corrected. Can you please drop this change from your PR? Once #961 is merged, the `go.mod` should not get out-of-sync any longer.